File: java2csharp/java/Samsung/ProfessionalAudio.FloatingConotrller.WidgetsLibrary/refactor/FcSapaActionDispatcher.java
package com.samsung.android.sdk.professionalaudio.widgets.refactor;

/**
 * @brief Interface to call remote actions using SAPA service
 */
public interface FcSapaActionDispatcher {

    /**
     * @brief Call action of given <code>actionId</code> using given instance of
     *        remote application
     *
     * @param instanceId    unique instance id
     * @param actionId      id of the action to call
     */
    void callAction(String instanceId, String actionId);

}

File: java2csharp/csharp/Samsung/ProfessionalAudio.FloatingConotrller.WidgetsLibrary/refactor/FcSapaActionDispatcher.cs
namespace com.samsung.android.sdk.professionalaudio.widgets.refactor
{

	/// <summary>
	/// @brief Interface to call remote actions using SAPA service
	/// </summary>
	public interface FcSapaActionDispatcher
	{

		/// <summary>
		/// @brief Call action of given <code>actionId</code> using given instance of
		///        remote application
		/// </summary>
		/// <param name="instanceId">    unique instance id </param>
		/// <param name="actionId">      id of the action to call </param>
		void callAction(string instanceId, string actionId);

	}

}